OpenImis.ePayment/Escape/Payment/Controllers/PaymentController.cs

```diff
@@ -117,6 +117,29 @@ public IActionResult GetReconciliation([FromBody] gepgSpReconcResp model)
                 var gepgFile = new GepgFoldersCreating("Reconc", "Data", reconc, env);
                 gepgFile.putToTargetFolderPayment();
 
+                foreach (var recon in model.ReconcTrxInf) 
+                {
+                    var paymentToCompare = imisPayment.GetPaymentToReconciliate(new { billId = recon.SpBillId, paidAmount = recon.PaidAmt });
+                    if (paymentToCompare != null)
+                    {
+                        int billId = (int) paymentToCompare.GetType().GetProperty("paymentStatus").GetValue(paymentToCompare);
+                        if (billId < 5)
```

Comment:
Please create a PaymentStatus static list with all statuses as constants and use them in the code for better understanding. As for GepgCodeResponses.

Reply:
ok!

Reply:
@dragos-dobre I created static class in similar way as for GePGCodeResponses. It should be the same like having static List. Then we have in code clear which payment status we are using.